migration/sql-files/062-workitem-related-changes.sql

```diff
@@ -0,0 +1,77 @@
+-- add a column to record the timestamp of the latest addition/change/removal of a comment on a workitem
+ALTER TABLE work_items ADD COLUMN commented_at timestamp with time zone;
```

Comment:
I already discussed about it with @alexeykazakov: the `updated_at` column is used by gorm to record the time of last update of the entity itself. Here the 2 new columns serve a different purpose.